build/manifest/main.go

```diff
@@ -101,7 +109,7 @@ func applyManifest(manifest *model.Manifest) error {
 	if manifest.HasServer() {
 		if err := ioutil.WriteFile(
 			"server/manifest.go",
-			[]byte(fmt.Sprintf(pluginIDGoFileTemplate, manifest.Id, manifest.Version)),
+			[]byte(fmt.Sprintf(pluginIDGoFileTemplate, manifest.Id, manifest.Version, model.ConfigToJsonWithoutEmptyFields(manifest.RequiredConfig))),
```

Comment:
Oh, I didn't realize we were using `ConfigToJsonWithoutEmptyFields` here as well. Comment added to original PR, but I'm wondering if we could just skip the call and let `model.ConfigFromJson` receive all the `nil` fields -- does that hurt anything?